CHANGES.md

```diff
@@ -38,9 +38,10 @@
 - Improved accessibility.
 - New "getOccupantActionButtons" hook, so that plugins can add actions on MUC occupants.
 - MUC occupants badges: displays short labels, with full label as title.
-- Fix: trying to use an emojis with an uppercase letter breaks the message field.
+- Fix: trying to use emojis with an uppercase letter breaks the message field.
 - Fix: renaming getEmojisByAtrribute to getEmojisByAttribute.
 - New config option [stanza_timeout](https://conversejs.org/docs/html/configuration.html#show-background)
+- Fix some typos in CHANGES
```

Comment:
Please remove this line. It's not necessary.

Reply:
Got it, I close this PR and create a new one